build.gradle

```diff
@@ -182,28 +183,29 @@ def generate_distro_tasks = { dist, gDescription, gPackageName, gInputFile, gVer
         suckUpEmptyDirectories('/var/lib/openhab2/persistence/rrd4j', user, permissionGroup, 0755)
         suckUpEmptyDirectories('/var/lib/openhab2/persistence/mapdb', user, permissionGroup, 0755)
         suckUpEmptyDirectories('/usr/share/openhab2/bin', user, permissionGroup, 0755)
-        from(debResourcesDir + 'etc/default/openhab2'){
+
+        from('resources/etc/default'){
           fileType CONFIG | NOREPLACE
           user 'root'
           permissionGroup 'root'
           fileMode 0644
           into '/etc/default'
         }
-        from(debResourcesDir + 'bin/setpermissions.sh'){
+        from('resources/bin/setpermissions.sh'){
```

Comment:
I've just had a look, it's an artifact from the conversion to it's own repo it would seem. It doesn't contain anything and isn't called. I should remove this.